test/rails_integration_test.rb

```diff
@@ -0,0 +1,29 @@
+require File.dirname(__FILE__) + "/test_helper"
+
+module Dummy
+  class Application < ::Rails::Application
+    config.eager_load = false
+  end
+end
+
+module ActiveRecord
+  class Base
+  end
+end
```

Comment:
right it's because of `rails` instead of `rails/all`. That's also why I have to require `action_controller/railties`, this test doesn't actually integrate against ActiveRecord, because ActiveRecord won't run without a bunch of config files, and this gem doesn't actually require any features of ActiveRecord, only that the Base class exists.